### PR TITLE
Ports examine boxes from tg and probably other servers too

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
https://github.com/tgstation/tgstation/pull/67937

examine blocks, we all love em, and now theyre comin to bee!

## Why It's Good For The Game
visauls look good :)

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: examine boxes are a comin
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/ai/examine.dm [hunk 1]
```diff
@@ -17,6 +17,10 @@
 			. += "The wireless networking light is blinking.\n"
 		else if (!shunted && !client)
 			. += "[src]Core.exe has stopped responding! NTOS is searching for a solution to the problem...\n"
-	. += "*---------*</span>"
+	. += "</span>"
 
 	. += ..()
+
+// no idea what this does but the original pr has it so
```
Comment (Penwin0) [inline code/modules/mob/living/silicon/ai/examine.dm]: Bad comment

File: code/__DEFINES/chat.dm [hunk 2]
```diff
@@ -19,3 +19,5 @@
 #define MESSAGE_TYPE_ADMINLOG "adminlog"
 #define MESSAGE_TYPE_ATTACKLOG "attacklog"
 #define MESSAGE_TYPE_DEBUG "debug"
+
```
Comment (Penwin0) [inline code/__DEFINES/chat.dm]:
```suggestion
/// Adds a generic box around whatever message you're sending in chat. Really makes things stand out.
```
You left this comment out from the original PR

**Review by Penwin0 — COMMENTED** [review]
I haven't gone through the whole thing yet

**Review by Penwin0 — COMMENTED** [review]
You missed changes to atoms.dm /atom/proc/examine(mob/user)

File: code/modules/mob/living/silicon/ai/examine.dm [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /mob/living/silicon/ai/examine(mob/user)
-	. = list("<span class='info'>*---------*\nThis is [icon2html(src, user)] <EM>[src]</EM>!")
```
Comment (Penwin0) [inline code/modules/mob/living/silicon/ai/examine.dm:2]: Is this span purposefully removed?

File: code/modules/mob/living/silicon/ai/examine.dm (hunk 1 above)
Comment (Penwin0) [inline code/modules/mob/living/silicon/ai/examine.dm]: It's still a bad comment even if you resolved the previous conversation

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Changes requested, see above

**Review by PowerfulBacon — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
Needs better PR body completion in future

File: tgui/packages/tgui-panel/styles/goon/chat-light.scss [hunk 4]
```diff
@@ -466,6 +466,13 @@ h1.alert, h2.alert		{color: #000000;}
 .text-normal {font-weight: normal; font-style: normal;}
 .hidden {display: none; visibility: hidden;}
 
+.examine_block {
+  background: #1b1c1e;
+  border: 1px solid #a4bad6;
+  margin: 0.5em;
+  padding: 0.5em 0.75em;
+}
```
Comment (PowerfulBacon) [inline tgui/packages/tgui-panel/styles/goon/chat-light.scss:474]: ... why would you make it black on light more

**Pull request merged**